capa/rules/__init__.py

```diff
@@ -456,7 +456,7 @@ def parse_feature(key: str):
 
 def parse_bytes(s: str) -> bytes:
     try:
-        b = codecs.decode(s.replace(" ", "").encode("ascii"), "hex")
+        b = bytes.fromhex(s)
```

Comment:
```suggestion
        b = bytes.fromhex(s.replace(" ", ""))
```

otherwise, looks good

can you please also remove the now unused codecs import?